terraform/modules/monitoring/metrics/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  env = [
+    "Serilog__MinimumLevel=${var.log_level}",
+    "ASPNETCORE_ENVIRONMENT=${var.dev_env}"
+  ]
```

Comment:
Should be given by the root module, in a `environment` variable. This would also reduce the number of variables to pass to each modules.